greenheart/tools/profast_tools.py

```diff
@@ -0,0 +1,288 @@
+        - price_breakdown.loc[
+            price_breakdown["Name"] == "One time capital incentive", "NPV"
+        ].tolist()[0]
+    )
+    remaining_financial = (
+        price_breakdown.loc[price_breakdown["Name"] == "Non-depreciable assets", "NPV"].tolist()[0]
+        + price_breakdown.loc[price_breakdown["Name"] == "Cash on hand reserve", "NPV"].tolist()[0]
+        + price_breakdown.loc[price_breakdown["Name"] == "Property insurance", "NPV"].tolist()[0]
+        - price_breakdown.loc[
+            price_breakdown["Name"] == "Sale of non-depreciable assets", "NPV"
+        ].tolist()[0]
+        - price_breakdown.loc[price_breakdown["Name"] == "Cash on hand recovery", "NPV"].tolist()[0]
+    )
+
+    if "capital_items" in config_keys:
+        capital_items = pf_config["capital_items"]
+        for item in capital_items:
+            key_name = f"{lco_str}: {item} ({lco_units})"
+            price_breakdown_capex[key_name] = (
+                price_breakdown.loc[price_breakdown["Name"] == item, "NPV"].tolist()[0]
+                + cap_expense * capex_fraction[item]
+            )
+        full_price_breakdown.update(price_breakdown_capex)
+
+    if "fixed_costs" in config_keys:
+        fixed_items = pf_config["fixed_costs"]
+        for item in fixed_items:
+            key_name = f"{lco_str}: {item} ({lco_units})"
+            price_breakdown_fixed_cost[key_name] = price_breakdown.loc[
+                price_breakdown["Name"] == item, "NPV"
+            ].tolist()[0]
+        full_price_breakdown.update(price_breakdown_fixed_cost)
+
+    if "feedstocks" in config_keys:
+        feedstock_items = pf_config["feedstocks"]
+        for item in feedstock_items:
+            key_name = f"{lco_str}: {item} ({lco_units})"
+            price_breakdown_feedstocks[key_name] = price_breakdown.loc[
+                price_breakdown["Name"] == item, "NPV"
+            ].tolist()[0]
+        full_price_breakdown.update(price_breakdown_feedstocks)
+
+    price_breakdown_taxes = (
+        price_breakdown.loc[price_breakdown["Name"] == "Income taxes payable", "NPV"].tolist()[0]
+        - price_breakdown.loc[price_breakdown["Name"] == "Monetized tax losses", "NPV"].tolist()[0]
+    )
+
+    if pf_config["params"]["general inflation rate"] > 0:
+        price_breakdown_taxes = (
+            price_breakdown_taxes
+            + price_breakdown.loc[
+                price_breakdown["Name"] == "Capital gains taxes payable", "NPV"
+            ].tolist()[0]
+        )
+
+    full_price_breakdown[f"{lco_str}: Taxes ({lco_units})"] = price_breakdown_taxes
+    full_price_breakdown[f"{lco_str}: Finances ({lco_units})"] = remaining_financial
+    lco_check = sum(list(full_price_breakdown.values()))
+    full_price_breakdown[f"{lco_str}: Total ({lco_units})"] = lco_check
+
+    return full_price_breakdown, lco_check
+
+
+def make_profast_capital_item(capex_usd, component, refurb=[0]):
```

Comment:
The rest of the functions in this file are not used I don't think and should probably be in a different PR. Also, we are somewhat tied with financial naming conventions. These functions require specific names, which concerns me and I would like to discuss more when we are ready to use it. Otherwise looks good.